packages/flutter/lib/src/widgets/animated_cross_fade.dart

```diff
@@ -206,6 +208,11 @@ class AnimatedCrossFade extends StatefulWidget {
   /// result in the widgets jumping about when the cross-fade state is changed.
   final AnimatedCrossFadeBuilder layoutBuilder;
 
+  /// Whether to exclude focus from the bottom child widget.
```

Comment:
Tiny nit: I'd add something like "When true, this is equivalent to wrapping the bottom widget with an [ExcludeFocus] widget while it is on the bottom of the cross-fade stack." Just so that there is a reference to `ExcludeFocus` in the docs for people to refer to so that they know the details of what that implies.

You could also talk about what happens when it's false instead, since true is the default.